app/js/controllers/relatedCtrl.js

```diff
@@ -3,10 +3,11 @@ KMCMenu.controller('relatedCtrl', ['$scope',
 		$scope.relatedOption = "relatedToEntry";
 		$scope.entryList = "";
 		$scope.playlistId = {id:'' ,text:''};
+		$scope.playerData.config.plugins["related"] = {disable: true};
 
 		// init radio buttons according to selected related entries source
-		if ($scope.playerData.config.player.plugins && $scope.playerData.config.player.plugins["related"]) {
-			var data = $scope.playerData.config.player.plugins["related"];
+		if ($scope.playerData.config.plugins && $scope.playerData.config.plugins["related"]) {
```

Comment:
can .plugins.related be undefined ? because in line 6 we call it without a check. so either we don't need an if or it needs to be above.
also I prefer doing
const data = $scope.playerData.config.plugins["related"];
before the if instead of accessing the field twice

Reply:
see now